app/jinja_filters.py

```diff
@@ -1,41 +1,48 @@
 # coding: utf-8
 import re
+from decimal import Decimal
+from typing import Any, Mapping, Optional, Union
 
 import flask
 import flask_babel
 from babel import numbers, units
 from flask import current_app
 from jinja2 import pass_eval_context
 from markupsafe import Markup, escape
+from wtforms import SelectFieldBase
 
 from app.questionnaire.rules.utils import parse_datetime
 from app.settings import MAX_NUMBER
 
 blueprint = flask.Blueprint("filters", __name__)
 
 
-def mark_safe(context, value):
+def mark_safe(context: Any, value: str) -> str:
     return Markup(value) if context.autoescape else value
 
 
-def strip_tags(value):
+def strip_tags(value: str) -> str:
     return escape(Markup(value).striptags())
 
 
 @blueprint.app_template_filter()
-def format_number(value):
+def format_number(value: int) -> str:
```

Comment:
This can accept both integers and floats as it uses `format_decimal()` further down which accepts floats as arguments.